proof/refine/ARM/Untyped_R.thy

```diff
@@ -3397,24 +3396,20 @@ lemma updateFreeIndex_updateCap_caps_no_overlap'':
   apply (clarsimp simp:caps_no_overlap''_def)
   apply (wp updateCap_ctes_of_wp)
   apply (clarsimp simp: modify_map_def ran_def cte_wp_at_ctes_of
-              simp del: atLeastAtMost_iff atLeastatMost_subset_iff atLeastLessThan_iff
-                        Int_atLeastAtMost atLeastatMost_empty_iff split_paired_Ex)
+              simp del: atLeastAtMost_simps)
   apply (case_tac "a = src")
-   apply (clarsimp simp del: atLeastAtMost_iff atLeastatMost_subset_iff atLeastLessThan_iff
-     Int_atLeastAtMost atLeastatMost_empty_iff split_paired_Ex)
+   apply (clarsimp simp del: atLeastAtMost_simps)
```

Comment:
This raises an issue for me, the amount of times that `atLeastAtMost_simps` is removed from the simpset: please tell me we're not the ones who put it in the simpset? If we did, it might be wise to take it out.

Reply:
No, I think all of the rules in `atLeastAtMost_simps` are added to the simpset within the HOL theory files

Reply:
Shame. If that's the case, then this is a fine way of handling the problem.